### ISSUE TITLE
Adding a button or command to update the player in their own configs as a special user or a check every time they enter the server so it updates to "VIP" or Special user.

### ISSUE THREAD

**Issue by PowderSol** [MarkusBordihn/BOs-Daily-Rewards#38]
**Is your feature request related to a problem? Please describe.**
Its related to players getting their special rewards from the special users feature, currently if you enable the player server-side, they get a somewhat buggy inventory to claim their rewards until they update their own daily-rewards-common file so they have the rewards too

**Describe the solution you'd like**
Maybe updating the player server-side only would be really hard to do, but a somewhat temporal fix would be so you made a command that added them client-side (in their own daily-rewards-common file) and they just had to restart the game to apply the config instead of adding themselves one by one in the configs.

**Describe alternatives you've considered**
Also considered as above running all the config files serverside, and they get their own daily rewards updated asap. but i'd have to guess that if it isnt in the mod yet, its because its too hard to do.

**Additional context**
Tried adding friends into the special rewards system, but it was annoying that i had to update their "client-side" configs everytime (even tho the file is a "common" file, meaning server-side)

**Comment by AnyankaT** [comment]
I agree with this. It would be great to have a way to update special rewards recipients without having to push a modpack update for all clients.
